es6-async-await/await.js

```diff
@@ -0,0 +1,56 @@
+import fetch from './fetch.js';
+
+// Keep track of app load time so each log message can be timed.
+// Log messages should all be approximately 1 second apart.
+const startTime = Date.now();
+const elapsed = () => `${Math.round((Date.now() - startTime) / 1000)}s -`;
+
+async function fetchOnce() {
+  try {
+    const response = await fetch('foo/bar.html');
+    const msg = await response.toString();
```

Comment:
This isn't necessary since `response` is already a string. Now, in the built-in browser function `fetch`, the result will be an object and you can get the message with `response.toString()`, but if you look at how `fetch` is defined in this exercise, it returns a string already.

Also, `String.toString()` is synchronous, so you don't need to `await` it. But once again, with the browser fetch, `Response.toString()` is asynchronous, so you _will_ need to `await` it.